code/modules/uplink/uplink_items.dm

```diff
@@ -1794,6 +1794,20 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	cost = 4
 	restricted_roles = list("Cook", "Botanist", "Clown", "Mime")
 
+/datum/uplink_item/role_restricted/strange_seeds
+	name = "Pack of strange seeds"
+	desc = "Mysterious seeds as strange as their name implies. Spooky."
+	item = /obj/item/seeds/random
```

Comment:
This will make strange seeds deconstructable for illegal tech. Please set the illegal_tech var for this uplink item FALSE.

Reply:
Oh, good catch. Imma do a fix tomorrow.

Reply:
Fixed.